libraries/loginPage.js

```diff
@@ -75,9 +60,9 @@ class Login {
     }
   }
 
-  async getEventlist() {
+  getEventlist() {
     try {
-      return await this.req.get(eventListUrl, {
+      return this.req.get(eventListUrl, {
```

Comment:
the catch inside this function will be useless then?

I'm putting out catch in this function to avoid polluting the caller with handling got http error